api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiPatient.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.Patient;
+
+/**
+ * Defines a MpiPatient in the system. A mpi patient is simply an extension of a patient and all that that
+ * implies.
+ */
+public class MpiPatient extends Patient {
+
+    private static final boolean mpiPatient = true;
+
+    /**
+     * Checks if patients is MpiPatient
+     * @return true if it is mpi patient, otherwise false
+     */
+    public boolean getMpiPatient() {
```

Comment:
In that case you should say "we can't use the conventional method name isMpiPatient() because the 'simplify' method used in the registrationapp module doesn't recognize that". Or better, fix that bug in the other module. :-)